### ISSUE TITLE
Other languages than English

### ISSUE THREAD

**Issue by galantra** [z1lc/AutoDefine#29]
Is it possible to modify this addon and use with other dictionaries' APIs? This would be great for languages other than English.

**Comment by z1lc** [comment]
I do not believe it is possible using the existing Merriam-Webster API (it's English-only), but I will happily review PRs that integrate with other services.
